src/components/dashboard/Claim.js

```diff
@@ -60,6 +73,12 @@ const Claim = props => {
   const wrappedGoodWallet = wrapper(goodWallet, store)
   const advanceClaimsCounter = useClaimCounter()
 
+  // calculate and format number of people who did claim today
+  const peopleClaimedToday = useMemo(() => {
+    const people = get(state, 'claimedToday.amount', 0)
+    return numeral(people).format('0a')
+  }, [state])
```

Comment:
pls
a) rename state/setState to something more exact. for example, claimState. Because setState correlates with React.Component's class method and could confuse other developers
b) destructurize amount from state above, add only the amount in the deps

Reply:
done